influxql/functions.go

```diff
@@ -464,7 +442,7 @@ func (r *FloatHoltWintersReducer) Emit() []FloatPoint {
 			}
 		}
 	} else {
-		l_0 += alpha * r.y[0]
+		l_0 += 0.5 * r.y[0]
```

Comment:
I think you should keep the constants for the defaults and use them here instead of a magic variable.